[2,"\n"]
[1,"\u001B[1mpandoc \u001B[22m\n  to: html\n  output-file: index.html\n  standalone: true\n  title-prefix: homepage\n  section-divs: true\n  html-math-method: mathjax\n  wrap: none\n  default-image-extension: png\n  css:\n    - styles.css\n  toc: true\n  \n\u001B[1mmetadata\u001B[22m\n  document-css: false\n  link-citations: true\n  date-format: long\n  lang: en\n  editor: visual\n  theme: cosmo\n  title: Faiz Essa\n  \n"]
[1,"Output created: _site/index.html\n\n\u001B[1m\u001B[34mPreparing to preview\u001B[39m\u001B[22m\n"]
[1,"\n\u001B[32mWatching files for changes\u001B[39m\n\u001B[32mBrowse at \u001B[39m\u001B[4m\u001B[32mhttp://localhost:3532/\u001B[39m\u001B[24m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[1mpandoc \u001B[22m\n  to: html\n  output-file: index.html\n  standalone: true\n  title-prefix: homepage\n  section-divs: true\n  html-math-method: mathjax\n  wrap: none\n  default-image-extension: png\n  css:\n    - styles.css\n  toc: true\n  \n\u001B[1mmetadata\u001B[22m\n  document-css: false\n  link-citations: true\n  date-format: long\n  lang: en\n  editor: visual\n  theme: cosmo\n  title: Faiz Essa\n  \n"]
[1,"Output created: _site/index.html\n\n\u001B[32mWatching files for changes\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[32mGET: /index.html\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///Applications/quarto/bin/quarto.js:42120:15)\n    at async file:///Applications/quarto/bin/quarto.js:82873:26\n    at async projectContext (file:///Applications/quarto/bin/quarto.js:82725:26)\n    at async refreshProjectConfig (file:///Applications/quarto/bin/quarto.js:102811:19)\n    at async file:///Applications/quarto/bin/quarto.js:102934:17\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///Applications/quarto/bin/quarto.js:42120:15)\n    at async file:///Applications/quarto/bin/quarto.js:82873:26\n    at async projectContext (file:///Applications/quarto/bin/quarto.js:82725:26)\n    at async refreshProjectConfig (file:///Applications/quarto/bin/quarto.js:102811:19)\n    at async file:///Applications/quarto/bin/quarto.js:102934:17\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///Applications/quarto/bin/quarto.js:42120:15)\n    at async file:///Applications/quarto/bin/quarto.js:82873:26\n    at async projectContext (file:///Applications/quarto/bin/quarto.js:82725:26)\n    at async refreshProjectConfig (file:///Applications/quarto/bin/quarto.js:102811:19)\n    at async file:///Applications/quarto/bin/quarto.js:102934:17\u001B[39m\n"]
[1,"\u001B[91mERROR: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n\nError: Project _quarto.yml validation failed.\n\nIn file _quarto.yml\n(line 6, columns 3--13) property name \u001B[34moutput-dir\u001B[91m is invalid\n5: \u001B[34m  title: \"homepage\"\u001B[91m\n6: \u001B[34m  output-dir: docs\u001B[91m\n    ~~~~~~~~~~~\n\u001B[34mℹ\u001B[91m The error happened in location \u001B[34mwebsite\u001B[91m:\u001B[34moutput-dir\u001B[91m.\n\n    at readAndValidateYamlFromFile (file:///Applications/quarto/bin/quarto.js:42120:15)\n    at async file:///Applications/quarto/bin/quarto.js:82873:26\n    at async projectContext (file:///Applications/quarto/bin/quarto.js:82725:26)\n    at async refreshProjectConfig (file:///Applications/quarto/bin/quarto.js:102811:19)\n    at async file:///Applications/quarto/bin/quarto.js:102934:17\u001B[39m\n"]
